### ISSUE TITLE
Make Navbar Responsive and Sticky for Improved Mobile Experience

### ISSUE THREAD

**Issue by AasthaSingh28** [Yash-Ainapure/dypcetclubs.live#9]
The current navbar lacks responsiveness in mobile view, affecting usability on smaller screens. Additionally, the navbar should be made sticky to improve accessibility during scrolling, as it currently disrupts the user experience. These changes will enhance the overall look and functionality of the navbar across all devices.

![Screenshot 2024-10-05 101816](https://github.com/user-attachments/assets/c66f1bcf-516d-407b-993b-c207c8c0f34b)
@Yash-Ainapure - Kindly assign me the issue so that I can work on this.

**Comment by Supreetpatel** [comment]
I want to work on these issue. Could u please assign it to me ?

**Comment by Yash-Ainapure** [comment]
> The current navbar lacks responsiveness in mobile view, affecting usability on smaller screens. Additionally, the navbar should be made sticky to improve accessibility during scrolling, as it currently disrupts the user experience. These changes will enhance the overall look and functionality of the navbar across all devices.
> 
> ![Screenshot 2024-10-05 101816](https://private-user-images.githubusercontent.com/138100998/373846131-c66f1bcf-516d-407b-993b-c207c8c0f34b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgxMDc0NDUsIm5iZiI6MTcyODEwNzE0NSwicGF0aCI6Ii8xMzgxMDA5OTgvMzczODQ2MTMxLWM2NmYxYmNmLTUxNmQtNDA3Yi05OTNiLWMyMDdjOGMwZjM0Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwNVQwNTQ1NDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NGZmYjMwZTdkYmFiOTI4MjAwZGI0Y2ZmMzA0NWJiYmU3OGQxNjAxMDMwNWQwMzA2OGYzY2M0MmFmY2M3NDE4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FG6r_xPZ74wnrRj8G9r0NDa2C1skaIcU7LRDwURucK0) @Yash-Ainapure - Kindly assign me the issue so that I can work on this.


I think this issue is same as #8

**Comment by AasthaSingh28** [comment]
I suggest making the navbar sticky or changing its background color, as it's not visible with a white background after scrolling. This can be paired with a scroll-to-top button for a professional look.

**Comment by Yash-Ainapure** [comment]
> I suggest making the navbar sticky or changing its background color, as it's not visible with a white background after scrolling. This can be paired with a scroll-to-top button for a professional look. @Yash-Ainapure

1. Can you suggest some good theme/color for the navbar, how will it look for our black N white themed website(at present)?
2.There isn't a need for scroll to top button.
If the navbar theme is good then we can move further.

**Comment by AasthaSingh28** [comment]
I will suggest for the #0B192C and #1E3E62 color, I think we can go with this as per theme. 
(Last two)
![Color Hunt Palette ff65001e3e620b192c000000](https://github.com/user-attachments/assets/e061ebb3-1122-4b11-97bc-b94dbe3cd4b9)

**Comment by Yash-Ainapure** [comment]
> I will suggest for the #0B192C and #1E3E62 color, I think we can go with this as per theme. (2nd and 3rd) @Yash-Ainapure ![Color Hunt Palette ff65001e3e620b192c000000](https://private-user-images.githubusercontent.com/138100998/373869895-e061ebb3-1122-4b11-97bc-b94dbe3cd4b9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgxODc1ODAsIm5iZiI6MTcyODE4NzI4MCwicGF0aCI6Ii8xMzgxMDA5OTgvMzczODY5ODk1LWUwNjFlYmIzLTExMjItNGIxMS05N2JjLWI5NGRiZTNjZDRiOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwNlQwNDAxMjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yYWJlNDE0NGIyYTc1ZDRjMzI1MDk0OGUxMWZhM2M5Y2EwNTIxMTY4MDc3MDYwZTZlZDhkNmYwZGEzYjEzMjg2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.XTr9ZDEAeSaXiiNZZT7xUdEVKfy9Ynqfmh6yiMxaSAw)

Sure, you can go ahead with the 2nd color option(faint blue one below orange).

**Comment by AasthaSingh28** [comment]
@Yash-Ainapure I wanted to ask that should I make Navbar sticky or not? And what about scroll to top button?

**Comment by Yash-Ainapure** [comment]
> @Yash-Ainapure I wanted to ask that should I make Navbar sticky or not? And what about scroll to top button?

Don't add scroll to top button and make navbar sticky

**Comment by AasthaSingh28** [comment]
@Yash-Ainapure I am trying to push the code but it is showing permission denied...Please check!
Permission to Yash-Ainapure/dypcetclubs.live.git denied to AasthaSingh28.

**Comment by Kuldeepsinghrajpoot** [comment]
can you assign me this task to resolve this issues

**Comment by AasthaSingh28** [comment]
@Yash-Ainapure I have successfully made the PR can you please merge it..
